README.md

```diff
@@ -47,6 +47,6 @@ This script is still in the "beta" stages and has been tested but not thoroughly
 
 The code is documented using JavaDoc/JSDoc style comments if you are interested in taking a peek at the source code.
 
-This script is adapted (but very different) from AutoBHH, originally by izchak and majorly revised by Raorn and Zarqon.
+This script is forked from RESPRiT. Which was adapted (but very different) from AutoBHH, originally by izchak and majorly revised by Raorn and Zarqon. Thank you all for your contributions to this script.
```

Comment:
we should link the original repo. once we merge this i am going to disconnect the fork so this in the readme will be the connection to the O.G. script